odl/solvers/advanced/proximal_operators.py

```diff
@@ -846,6 +898,61 @@ def proximal_cconj_l1(space, lam=1, g=None, isotropic=False):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    The :math:`L_1`-norm :math:`F` is the functional
+
+        .. math::
+
+            F(x) = \\lambda ||x - g||_1.
+
+    The convex conjugate :math:`F^*` of :math:`F` is given by the indicator
+    function of the set :math:`box(\\lambda)`
```

Comment:
This won't look pretty. I'd replace `box` by a letter and use `lambda` as subscript. The letter `Q` is frequently used for boxes, so that would be my choice.